CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+## v0.5.2
+
+This is the same as v0.5.1, but addresses a CI issue that stopped the creation of the Windows installer executable. 
+
+#### Windows Limitations (beta notice)
```

Comment:
Did you intend to duplicate this section with the one in v0.5.1 below?

Reply:
@alfred-landrum: The duplication was intentional. This speaks to a style question you're certainly welcomed to weigh in on.

These limitations are technically in every Windows release until we address them. So if we look at these as Release Notes, they should be documented alongside every release. Now, due to the level of detail involved, I also respect that extending what I've done here means there's going to be bulky redundancy when looking at the changelog or scrolling down on the GitHub Releases tab.

I'm open to suggestion on ways to improve on that. One thought is that I could break out the detailed text into something like a `README-Windows.md` and just link to that from each block of release note bullets. That would significantly reduce the redundant visual crowding.

Reply:
👍 I like the idea of breaking the notes out to a separate windows-specific file.

Reply:
@alfred-landrum: I've taken a shot at breaking it out into a separate file and linking to it from the `CHANGELOG.md`. Since the one note for v0.5.2 says it's effectively "the same as v0.5.1", I didn't repeat a bullet there about Windows support. But if the approach here is ok with you, in releases going forward, I'd be in favor of including a one-liner bullet:

* Windows releases are currently unsigned (unlike our Mac releases). See README-Windows for details.

It's still a bit of redundancy, but as long as we're coupling a main use case to Wireshark and 70-90% of Wireshark users are on Windows, I'd feel better doing all we can to try and set expectations.